tests/Unit/Schema/Directives/Args/RulesDirectiveTest.php

```diff
@@ -26,17 +26,15 @@ public function itCanValidateQueryRootFieldArguments()
                 [
                     'path' => ['foo'],
                     'locations' => [['line' => 3, 'column' => 13]],
-                    'message' => 'Validation failed for the field [foo]',
+                    'message' => 'Validation failed for the field [foo].',
                     'category' => 'validation',
-                    'extensions' => [
-                        'validation' => [
-                            'bar' => [
-                                'The bar field is required.'
-                            ]
-                        ]
-                    ]
+                    'validation' => [
```

Comment:
Let's keep the extra messages under extensions always, according to the spec recommendation.